### ISSUE TITLE
Mac OS media buttons to launch and control player

### ISSUE THREAD

**Issue by jeanrajotte** [clementine-player/Clementine#6384]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [x] I checked the issue tracker for similar issues
- [x] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [x] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System: Mac OS Mojave 10.14.6 (18G87)
- Clementine version: 1.3.1-611-gda04471

### Expected behaviour / actual behaviour

Pressing play/pause to launch, then play or pause. Next and prev to move between tracks/channels

### Steps to reproduce the problem (only for bugs)

Download the latest and install on top of existing.
The keys are no longer mapped. They launch iTunes instead.

No help online about this. 3rd party apps like "magic keys" only launch Clementine, they don't control playback.

Help please!!

Plus, it would be great if something like this were documented.

It was working until I installed the latest.  :(

**Comment by maxisaninja** [comment]
This is still broken in Mac OS as of 1.4rc1 and the most recent build, clementine-1.4.0rc1-99-g237a2fe8e.dmg.

I've been using Clementine 1.3.1-240-g1a2f6e2, which is a version I settled on as media key functionality works in this version but it doesn't have an unrelated and troublesome sleep/wake bug either.

I've tested versions 1.31-240, 242, and 243 and the media key function works in all of them. 260 has a broken dmg, but the media keys stop working for the first time in in 261 so something changed between 243 and 261 to break them.

Of course, installing Catalina means that the media keys no longer act even on these older versions of clementine unless clementine is the app in front, the media keys when any other app is in focus open the absolutely terrible Apple Music app. So there are likely two interconnected issues here. 

A possible workaround would be providing hooks for keyboard shortcuts so users can use function keys to dodge these media key related issues that have appeared in Catalina from Apple's attempts to cram all users into its walled garden music ecosystem.
